### PR TITLE
Update cpmr0024.md explaining the prerelease tag should be in the version

### PR DESCRIPTION
**Pull request by tunisiano187**
Explain the solution to cpmr0024

## Description Of Changes
Explain that the prerelease tag should be in the version

## Motivation and Context
cpmr0024 page was a template, not a complete explanation

## Change Types Made
* [ ] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [X] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left-hand side)/
* [ ] Menu structure has been updated

## Related Issue

[Related Issue](https://github.com/chocolatey/docs/issues/990)

### CODE REVIEW

File: input/en-us/community-repository/moderation/package-validator/rules/cpmr0024.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ RuleType: Requirement
 
 ## Issue
 
-In the nuspec,
+In the nuspec, the id contains a prerelease tag, that tag should be located in the version instead
```
Comment (pauby) [inline input/en-us/community-repository/moderation/package-validator/rules/cpmr0024.md:17]:
```suggestion
The `<id>` field in the package `.nuspec` file, contains a prerelease version name such as `alpha`, `beta`, etc.
```

File: input/en-us/community-repository/moderation/package-validator/rules/cpmr0024.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ RuleType: Requirement
...
 
 ## Recommended Solution
 
-Please update _ so that _
+Please update the nuspec so that the id only contains the real name, and add the prerelease part to the version, that will be added as prerelease on chocolatey and made available through the **--pre** parameter on the **choco install** command
```
Comment (pauby) [inline input/en-us/community-repository/moderation/package-validator/rules/cpmr0024.md:21]:
```suggestion
Please update the `<id>` field in the `.nuspec` file to ensure it only contains the name. The prerelease version name should be added to the `<version>` field. Prerelease packages can be [installed](https://docs.chocolatey.org/en-us/choco/commands/install#options-and-switches) or [upgraded](https://docs.chocolatey.org/en-us/choco/commands/upgrade#options-and-switches) using the `--pre` option in Chocolatey CLI.
```

**Review by pauby — CHANGES_REQUESTED** [review]
@tunisiano187 thank you very much for submitting this PR! I just wanted to suggest a few changes to make it clearer.

**Review by tunisiano187 — COMMENTED** [review]
Corrected, thank you Paul

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
Thanks @tunisiano187 for the docs update! I've rebased it and merged.

**Pull request merged**